modules/dnn/src/webnn/README.md

```diff
@@ -0,0 +1,11 @@
+## Build Instructions
+
+### Build WebNN-native and set the environment variable
+
+Refer to [WebNN's build instructions](https://github.com/webmachinelearning/webnn-native) to complete the build of WebNN-native.
+
+Set environment variable `WEBNN_NATIVE_DIR` to enable native DNN_BACKEND_WEBNN build: `export WEBNN_NATIVE_DIR=${PATH_TO_WebNN}`. Please let `WEBNN_NATIVE_DIR` points the output directory of webnn-native build (e.g. webnn-native/out/Release).
+
+### Test native DNN_BACKEND_WEBNN backend
+Add -DWITH_WEBNN=ON to the cmake command to build the WebNN module such as:
+`cmake -DWITH_WEBNN=ON ../opencv` (according to the [Installation in Linux](https://docs.opencv.org/master/d7/d9f/tutorial_linux_install.html))
```

Comment:
> https://docs.opencv.org/master

Direct URL links on docs.opencv.org are not allowed.
Use doxygen references instead.

Reply:
Hi @alalek, I changed this link and I'm not sure the new link is as your expectation, so please have a look. Thanks!

Reply:
Please use `@ref tutorial_linux_install`

Reply:
Thanks! I have modified this in the new commit.